src/buttons/Chip.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  TouchableOpacityProps,
+  TouchableNativeFeedbackProps,
+  StyleProp,
+  ViewStyle,
+  TextStyle,
+} from 'react-native';
+import { withTheme } from '../config';
+import Button from './Button';
+import { IconNode } from '../icons/Icon';
+import { Theme } from '../config/theme';
+import { TextProps } from '../text/Text';
+
+export type ChipProps = TouchableOpacityProps &
```

Comment:
Inherit from the `ButtonProps` instead of rebuilding.

Reply:
Should I still export ChipProps or just keep the ButtonProps and pass that into the Chip component?

Reply:
Sure thing! Working on it!

Reply:
Done!